docs/changes/coordinates/14354.api.rst

```diff
@@ -0,0 +1,3 @@
+``get_moon()`` is deprecated and may be removed in a future version of
+``astropy``. The geocentric coordinates of the Moon can be obtained by calling
+``get_body("moon")``.
```

Comment:
```suggestion
``get_body("moon", time, [location])``.
```

Reply:
I've changed the phrasing of the change log entry, although I tried to make the example function calls less shell-like and more Python-like.